CONTRIBUTING.md

```diff
@@ -12,7 +12,35 @@ We hold regular meetings which you can see [here](https://github.com/open-featur
 
 We are also present on the `#openfeature` channel in the [CNCF slack](https://slack.cncf.io/).
 
-## Project Governance
+## Project governance
 
-Project governance is documented [here](https://github.com/open-feature/governance).
-We plan to have open and vendor neutral governance in the project once it is fully established.
+Project governance is documented [here](https://github.com/open-feature/community/blob/main/governance-charter.md).
+
+## Repository requirements
+
+We recommend repositories in the project adhere to some security and maintenance guidelines. These are _only guidelines_, and may not be applicable or valuable in all instances. Adherence to these guidelines may be required for 1.0 artifact releases, at the discretion of the [Technical Steering Committee](https://github.com/open-feature/community/blob/main/governance-charter.md#technical-steering-committee-tsc) and maintainers.
+
+| Requirement                    | Recommended solution(s)                                                                                                                   |
+| ------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| artifact and image signing     | language-specific tools, [cosign][cosign]                                                                                                 |
+| automated publishing           | github actions (with permissions applying principle of least privilege), language-specific tools (Maven, nuget, NPM, etc)                 |
+| container, base image scanning | [snyk][snyk], [trivy][trivy]                                                                                                              |
+| code ownership                 | branch protection rules and CODEOWNERS files                                                                                              |
```

Comment:
I think just having `branch protection rules` is too vague. If we want to suggest code review from someone other than the author then I think we should spell that out explicitly.

Reply:
I will flesh-out this section a bit more.

Reply:
I've added a footnote about this.